react/features/toolbox/components/ToolbarButton.web.js

```diff
@@ -214,11 +217,11 @@ class ToolbarButton extends AbstractToolbarButton {
 
             if (!button.unclickable) {
                 if (button.tooltipText) {
-                    UIUtil.setTooltipText(this.button,
+                    setTooltipText(this.button,
                         button.tooltipText,
                         tooltipPosition);
                 } else {
-                    UIUtil.setTooltip(this.button,
+                    setTooltip(this.button,
```

Comment:
I started this as a part of the effort to reduce the size of the always on top bundle. But at the the problem with UIUtils turned out to be related to an unused import. I don't think I need this change but at the end I liked how the tooltips are separated from all other random util methods.

I think I can easily revert change if needed?

I was not aware of the tooltips reactification but I don't see a problem with this. Are you worried that this changes are going to be in conflict with the reactification PR?